.travis.yml

```diff
@@ -8,18 +8,19 @@ matrix:
   fast_finish: true
   include:
     # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
-    - {env: DJANGO=2.1, python: '3.5'}
-    - {env: DJANGO=2.1, python: '3.6'}
-    - {env: DJANGO=2.1, python: '3.7'}
-    - {env: DJANGO=2.1, python: 'nightly'}
-
     - {env: DJANGO=2.2, python: '3.5'}
     - {env: DJANGO=2.2, python: '3.6'}
     - {env: DJANGO=2.2, python: '3.7'}
     - {env: DJANGO=2.2, python: 'nightly'}
```

Comment:
We could also test 2.2 on 3.8

Reply:
I did not see that Django 2.2 supports python 3.8 since 2.2.8.
I'll add it.